stickytimelineview/src/main/java/xyz/sangcomz/stickytimelineview/TimeLineRecyclerView.kt

```diff
@@ -65,12 +67,29 @@ class TimeLineRecyclerView(context: Context, attrs: AttributeSet?) : RecyclerVie
      * Add RecyclerSectionItemDecoration for Sticky TimeLineView
      *
      * @param callback SectionCallback
+     * if you'd like to know more mode , look at res/values/attrs.xml
      */
     fun addItemDecoration(callback: RecyclerSectionItemDecoration.SectionCallback) {
         recyclerViewAttr?.let {
-            this.addItemDecoration(RecyclerSectionItemDecoration(context,
-                    callback,
-                    it))
+            val decoration: ItemDecoration =
+                    when (it.mode) {
+                        0x00 -> { // vertical
```

Comment:
```    
companion object{
        private const val VERTICAL = 0x00 
        private const val HORIZONTAL = 0x01 
    }
```
How about declaring it as a constant like this?
Then you can remove unnecessary comments.